scripts/build.sh

```diff
@@ -0,0 +1,293 @@
+#!/bin/bash
+
```

Comment:
I started using `PS4` to show transparency to users on which command we're running. I'd like to know if you think this is useful.

```bash
PS4='\033[1m[$0:${LINENO}] $\033[0m '
set -ex
set -o pipefail
```

Reply:
Cool idea 👍